ui: CSS modules for PlanView component

koorosh opened the pull request:
Depends on: #47484
Related to: #47527

- Refactored font imports to correctly resolve paths when module is required from different locations;
Fonts are imported directly from `app.styl` file which allows import `typography.styl` without dependencies. This change was required because importing `typography.styl` file from CSS modules
failed with unresolved paths inside of `fonts.styl` file (which was required in `typography.styl` file).
Before,
```
app.styl
|-- typography.styl
     |-- fonts.styl
```
Now:
```
app.styl
|-- typography.styl
|-- fonts.styl
```
- Move all files related to PlanView component under `planView` directory
- Added storybook for `PlanView` component
- `planView.module.styl` file contains copy of styles (from `statements.styl`) which is used by component only.

Release note: None

koorosh committed:
ui: Add Statements page storybook

To test styles isolation for statements page
we need storybook which displays entire
Statements screen only.

To make it work, RouterProvider decorator
is added which connects router to dummy (empty)
store.

`statementsPage.fixture.ts` file contains snapshot
of required props for StatementsPage component

Release note: None

koorosh committed:
ui: CSS modules for PlanView component

- refactor fonts imports to correctly resolve paths
when module is required from different locations;
- move all files related to PlanView component under
`planView` directory
- Added story for PlanView component

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/47513)
<!-- Reviewable:end -->

koorosh committed:
ui: Refactor classNames usage with bind for PlanView component

Previously, class names were constructed by simply accessing
style modules class names and assigning it to classes. It was
cumbersome and not readable at all.

To enhance this, `classnames/bind` alternate is used, which
allows simply put class names.

Release note: None

koorosh commented:
bors r+
